src/lib/learn-client/api/tutorial/index.ts

```diff
@@ -0,0 +1,83 @@
+import path from 'path'
+import {
+  getAllTutorialsOptions,
+  identifier,
+  Tutorial,
+  TutorialFullCollectionCtx,
+} from 'lib/learn-client/types'
+import { get, toError } from '../../index'
+import { formatBatchQueryStr, formatIdentifier, fetchAll } from '../utils'
+import { formatTutorialData } from './formatting'
+import { augmentTutorial } from './augment-tutorial'
+import { ApiTutorial } from 'lib/learn-client/api/api-types'
+
+export const TUTORIAL_API_ROUTE = '/tutorials'
+
+export async function getTutorial(
+  idOrSlug: identifier
+): Promise<TutorialFullCollectionCtx | null> {
+  const identifier = formatIdentifier(idOrSlug)
+
+  // /tutorials/:id
+  const route = path.join(TUTORIAL_API_ROUTE, identifier)
+  const getTutorialRes = await get(route)
+
+  if (getTutorialRes.ok) {
+    const res = await getTutorialRes.json()
+    const augmentedData = await augmentTutorial(res.result) // additional Api call here
+    return formatTutorialData(augmentedData) as TutorialFullCollectionCtx
```

Comment:
This is something we may adjust, to not return the full collection context here. Waiting to do this until I get a better idea of the data needs.